module_set.go

```diff
@@ -0,0 +1,30 @@
+package zgrab2
+
+// ModuleSet is a container holding named scan modules. It is a wrapper around a
+// map.
+type ModuleSet map[string]ScanModule
```

Comment:
If there is ever going to be a possible writer and reader that access the map concurrently, ModuleSet should be a struct that contains a`sync.Mutex` to guard the map.  e.g.,
```go
type ModuleSet struct {
    lock sync.Mutex // protects Modules
    Modules map[string]ScanModule
}
```
And then of course the methods can all do s.Lock() / s.Unlock() as appropriate.